lib/sedna/core/joint_inference/joint_inference.py

```
@@ -106,6 +106,29 @@ def __init__(self, estimator=None, config=None):
             self.estimator.load(self.model_path)
         self.cloud = ModelClient(service_name=self.job_name,
                                  host=self.remote_ip, port=self.port)
+        hem = self.get_parameters("HEM_NAME")
+        hem_parameters = self.get_parameters("HEM_PARAMETERS")
+
+        try:
+            hem_parameters = json.loads(hem_parameters)
+            if isinstance(hem_parameters, (list, tuple)):
+                if isinstance(hem_parameters[0], dict):
+                    hem_parameters = {
+                        p["key"]: p.get("value", "")
+                        for p in hem_parameters if "key" in p
+                    }
+                else:
+                    hem_parameters = dict(hem_parameters)
+        except Exception:
+            hem_parameters = None
```

if Exception, need to warn it instead of swallow it